src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue

```diff
@@ -92,6 +116,123 @@ export default class DOWLandingPage extends Vue {
     }
   ];
 
+  get doesCurrentEnvExist(): boolean {
+    return CurrentEnvironment.currentEnvironment && 
+            CurrentEnvironment.currentEnvironment.current_environment_exists === "YES";
+  }
+
+  /**
+   * Sets the 'label' and 'isComplete' properties of the optimize/replicate section.
+   *
+   * Label gets built using the classification level options that the user selects. Default label
+   * is left as is, if no levels are selected
+   *
+   * Current functions are complete, if all the below are true;
+   * 1. If the user chooses NOT to optimize or replicate
+   * 2. If the user chooses to optimize or replicate AND
+   *      chooses (NO) or (YES for additional growth and enters anticipated capacity)
+   * 3. If the user chooses to optimize or replicate AND
+   *      chooses (NO) or (YES for phased approach and enters approach schedule)
+   */
+  setCurrentFunctionsProperties(): void {
```

Comment:
console js errors thrown if user selected "No" for current environment (or no for current contract which would leave current environment as empty string) in step 4.